src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -713,7 +713,11 @@ export class CategoryFacet extends Component implements IAutoLayoutAdjustableIns
   private buildMoreButton() {
     const svgContainer = $$('span', { className: 'coveo-facet-more-icon' }, SVGIcons.icons.arrowDown).el;
     SVGDom.addClassToSVGInContainer(svgContainer, 'coveo-facet-more-icon-svg');
-    const more = $$('div', { className: 'coveo-category-facet-more', tabindex: 0 }, svgContainer);
+    const more = $$(
```

Comment:
Would it be more consistent to use `new AccessibleButton()` here? Similarly for the `less` button.